docs: add auth-user-session-idle-ttl-minutes setting (#1653)
rancher/rancher-docs: docs/api/api-tokens.md

Time to live (TTL) duration in minutes, used to determine when a user auth session token expires. When expired, the user must log in and obtain a new token. This setting is not affected by [`auth-token-max-ttl-minutes`](#auth-token-max-ttl-minutes). Session tokens are created when a user logs into Rancher.

### auth-user-session-idle-ttl-minutes

> Time to live (TTL) without user activity for tokens controlling a login session, in minutes.
> By default, the value for [`auth-user-session-idle-ttl-minutes`](#auth-user-session-idle-ttl-minutes) is be set to the same value as [`auth-user-session-ttl-minutes`](#auth-user-session-ttl-minutes) (for backward compatibility reasons), and it must never be greater than this value.

Reviewer: What about this?

```suggestion
Time to live (TTL) without user activity for login sessions tokens, in minutes.
By default, [`auth-user-session-idle-ttl-minutes`](#auth-user-session-idle-ttl-minutes) is set to the same value as [`auth-user-session-ttl-minutes`](#auth-user-session-ttl-minutes) (for backward compatibility). It must never exceed the value of `auth-user-session-ttl-minutes`.
```
